packages/quantic/force-app/main/default/lwc/quanticSmartSnippet/quanticSmartSnippet.js

```diff
@@ -127,8 +125,9 @@ export default class QuanticSmartSnippet extends LightningElement {
   }
 
   updateSearchStatusState() {
-    this.feedbackSubmitted = false;
-    this.shouldBeInNeutralState = true;
+    if (!this.state.disliked && !this.state.liked) {
```

Comment:
So this state would be reset correctly already in headless on each new query unless the answer is the same?